api/app/serializers/spree/v2/storefront/user_serializer.rb

```diff
@@ -4,7 +4,7 @@ module Storefront
       class UserSerializer < BaseSerializer
         set_type :user
 
-        attributes :email
+        attributes :email, :public_metadata, :private_metadata
```

Comment:
@aleksandarpetrushev we don't want to expose `private_metadata` only public, however, we need a way to store private metadata via storefront api